_includes/articles/payment-batching.md

```diff
@@ -156,9 +153,10 @@ transaction group containing your transaction.
 
 The closer to a limit a transaction group becomes, the less likely
 you'll be able to fee bump your transaction using either
-Child-Pays-for-Parent (CPFP) fee bumping or Replace-by-Fee (RBF) fee
+[Child-Pays-for-Parent (CPFP)][topic cpfp] fee bumping or [Replace-by-Fee
+(RBF)][topic rbf] fee
 bumping.  In addition, the more unconfirmed children a transaction has,
-the more RBF fee bumping will cost as you'll have to pay for both the
+the more RBF fee bumping costs because you'll have to pay for both the
 increased feerate of your transaction as well as for all the potential
 fees lost to miners when they remove any child transactions in order
 to accept your replacement.
```

Comment:
Remove "Note that" from the paragraph below.

Reply:
Is this a duplicate of https://github.com/bitcoinops/bitcoinops.github.io/pull/461/files/d043fdf182a85e6fddcdddaf308f9e67396c37dc#r555014652 ?  I don't see a "Note that" in the paragraph below.

Reply:
No, this is referring to the paragraph beginning "Note that these problems are not unique to batched payments".